### ISSUE TITLE
why LOSS is Negative?

### ISSUE THREAD

**Issue by aghaee1367** [kaituoxu/Conv-TasNet#42]
Hi, 
the loss function is usually positive, why is it negative here?

**Comment by Autism2020** [comment]
the code use si_snr as the criterion. loss  =   -1 * si_snr . when the loss is negative, the snr is positive.

**Comment by choresefree** [comment]
> the code use si_snr as the criterion. loss = -1 * si_snr . when the loss is negative, the snr is positive.

sry, i am fresh hand, will it hurt training? thanks!
